src/assets/scss/app.scss

```diff
@@ -252,6 +253,16 @@ a {
   }
 }
 
+// The flatPickr input is readonly by default, but we do not want to style it as
```

Comment:
does it work by keyboard input if i tab to it though?

Reply:
Right now it doesn't, which is the flatPickr default, and I think the user would have to type something goofy like "2019/01/01 to 2019/01/31". But flatPickr does seem to support direct keyboard input. I'm happy to configure that if that'd be useful!

Reply:
yes, please. keyboard input is important.

Reply:
I looked more into this, and it turns out that the default configuration actually does support keyboard navigation through the calendar: if you tab to the input, the calendar will open, then you can select a range using the arrow keys and Enter.

Reply:
So I think it should work as-is.